docs/porting/libraries.md

```diff
@@ -1,5 +1,224 @@
+
+1. File | New Project.
+2. Select ".NET Core" under Visual C#.
+3. Selecing a template with the xproj icon. 
+
+### When to pick csproj
+
+You can target .NET Core with the traditional project system in Visual Studio, `csproj`, by creating a Portable Class Library (PCL) and selecting ".NET Core" in the project configuration dialog.
+
+If you have more intricate project system needs, this should be your choice.  It may also be the best choice if you're mixing .NET Core and .NET Framework code in a larger solution.  Note that if you wish to multitarget by generating platform-specific assemblies like with the `xproj` project system, you'll need to create a "Bait and Switch" PCL, [as described here](https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/).
+
+## Porting your tests (if applicable)
+
+The best way to make sure everything works when you've ported your code is to *test your code as you port it to .NET Core*.  To do this, you'll need to use a testing framework that will build and run tests for .NET Core.  Currently, you have two options:
+
+* [xUnit](https://xunit.github.io/)
+   - [Getting Started](http://xunit.github.io/docs/getting-started-dnx.html)
+   - [Tool to convert an MSTest project to xUnit](https://github.com/dotnet/codeformatter/tree/master/src/XUnitConverter)
+* [NUnit](http://www.nunit.org/)
+  - [Getting Started](https://github.com/nunit/docs/wiki/Installation)
+  - [Blog post about migrating from MSTest to NUnit](http://www.florian-rappl.de/News/Page/275/convert-mstest-to-nunit)
+  
+Currently, MSTest does not run on .NET Core.  If you're using MSTest, you'll have to migrate.  If you've been using Xunit or Nunit to write your tests for your library, great!
+
+## Recommended approach to porting
+
+Finally, porting the code itself!  Ultimately, the actual porting effort will depending heavily on how your .NET Framework code is structured.  That being said, here is a recommended approach which may work well with your codebase.
+
+A good way to port your code is to begin with the "base" of your library.  This may be data models or some other foundational classes and methods that everything else uses directly or indirectly.
+
+1. Copy over the "base" of your library into the new .NET Core project and select the version of the .NET Platform Standard you wish to support.
+2. Make any changes needed to get the code to compile.
+
+   a. Much of this may just require adding NuGet package dependencies to your `project.json`.
+   
+   b. It may helpful to comment out old code as you slowly get things compiling.
+
+3. Run tests and make any needed adjustments.
+   
+4. Pick the next layer of code to port over and repeat steps 2 and 3!
+
+If you methodically move "out" from the "base" of your library, testing each layer as needed, you shouldn't run into unexpected issues.
```

Comment:
I wouldn't say "you shouldn't run into unexpected issues", because Murphy's Law, right?  maybe something more like "you break down the porting job more systematically, thus localizing the problems to one layer of code at a time"